examples/ECCX08HMAC/ECCX08HMAC.ino

```diff
@@ -0,0 +1,76 @@
+/*
+    ECCX08 HMAC functionality example
+
+    This sketch uses the ECC608 to generate an hmac on some data.
+    Stores key using nonce.
+
+    Used the Arduino Nano RP2040.
+
+    created 10 October 2022
+    by Raul Leclair
+*/
+
+#include <ArduinoECCX08.h>
+
+byte nonceKey[] = {
+    0x10, 0x10, 0x10, 0x10
+};
+
+byte data[] = {
+    0x10, 0x10, 0x10, 0x10, 0x10, 0x10, 0x10, 0x10,
+    0x10, 0x10, 0x10, 0x10, 0x10, 0x10, 0x10, 0x10,
+    0x10, 0x10, 0x10, 0x10, 0x10, 0x10, 0x10, 0x10,
+    0x10, 0x10, 0x10, 0x10, 0x10, 0x10, 0x10, 0x10,
+    0x10, 0x10, 0x10, 0x10, 0x10, 0x10, 0x10, 0x10,
+    0x10, 0x10, 0x10, 0x10, 0x10, 0x10, 0x10, 0x10,
+    0x10, 0x10, 0x10, 0x10, 0x10, 0x10, 0x10, 0x10,
+    0x10, 0x10, 0x10, 0x10, 0x10, 0x10, 0x10, 0x10,
+    0x10, 0x10, 0x10, 0x10, 0x10, 0x10, 0x10, 0x10,
+};
+
+int dataLength = 72;
+
+void setup() {
+    Serial.begin(115200);
+    while (!Serial);
+
+    if (!ECCX08.begin()) {
+        Serial.println("Failed to initialize ECC608 board.");
+        while (1);
+    }
+
+    // Perform nonce
+    if (!ECCX08.nonce(nonceKey))
+    {
+        Serial.println("Failed to do nonce.");
+        while (1);
+    }
+
+    // Starting HMAC operation on tempkey slot
+    if (!ECCX08.beginHMAC(0xFFFF)) {
```

Comment:
Might be useful to make `0xFFFF` a `#define` and give it a descriptive name like `TEMPKEY_SLOT_INDEX` or something like that.

Reply:
Resolved, added `#define TEMPKEY_SLOT 0xFFFF`